BMS Workspace-git/bms-1227-fw/Phantom_drivers/include/bms_data.h

```diff
@@ -9,23 +9,48 @@
 
 #include "sys_common.h"
 
+typedef enum State_t
+{
+    BMS_RUNNING,
+    BMS_CHARGING,
+    BMS_FAULT
+}State_t;
```

Comment:
Name of the struct is vague, have it specified that it's related to BMS

Reply:
Changed to BMSState_t